api-module-library/slack/test/api.test.js

```diff
@@ -85,6 +85,12 @@ describe(`Should fully test the ${config.label} API Class`, () => {
                 });
                 expect(history.ok).toBeTruthy();
             });
+            it('should return channel members', async () => {
+                const history = await api.getChannelMembers({
```

Comment:
Keep naming consistent. Instead of `history` should be something like `members`.